From 6eafbb943365810ead11bed3d9ab18b3365dd0db Mon Sep 17 00:00:00 2001
From: Krishnamurthy D V <kmurthy@netlogicmicro.com>
Date: Mon, 26 Dec 2011 01:40:40 -0800
Subject: [PATCH 0276/1532] openssl (sae-rx-vc) should be shared with nae-vc 0,
 as msgring interrupt handler and napi handler is looking only for vcs 0 and
 1.

[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/storm-linux.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/boot/dts/storm-linux.dts b/arch/mips/boot/dts/storm-linux.dts
index 71bcc1f..8d1ae4e 100644
--- a/arch/mips/boot/dts/storm-linux.dts
+++ b/arch/mips/boot/dts/storm-linux.dts
@@ -31,7 +31,7 @@
                                 nae-rx-vc = <0>;
                                 nae-fb-vc = <1>;
 				napi-vc-mask = <0x3>;
-				sae-rx-vc = <2>; /*  */
+				sae-rx-vc = <0>; /*  */
 				sae-rx-sync-vc  = <3>; /* should be exclusive */
 				ipsec-async-vc  = <0>;
                                 ipsec-sync-vc   = <2>;
-- 
1.9.1

